test/data/experimental/dummy_doc.yaml

```diff
@@ -0,0 +1,177 @@
+---
+## Document with content + optional layout info
+description: { } # DescriptionType - TBD
+name: "dummy_doc"
+origin:
+  binary_hash: 7954723514066505909
+  filename: dummy_doc.pdf
+  mimetype: application/pdf
+  uri: null
+
+# Root element for any headers, footers, framing, navigation elements, all other non-body text, type GroupItem
+furniture:
+  name: "_root_"
+  self_ref: "#/furniture"
+  parent: null # Only root elements have no parent.
+  children: # only the first-level children appear here, as references (RefItem)
+    - $ref: "/texts/0"
+
+# Root element for anything in the document body, type GroupItem
+body:
+  name: "_root_"
+  self_ref: "#/body"
+  parent: null # Only root elements have no parent.
+  children: # only the first-level children appear here, as references (RefItem)
+    - $ref: "/texts/1"
+    - $ref: "/pictures/0"
+    - $ref: "/texts/3"
+    - $ref: "/tables/0"
+
+# All groups of items nested deeper in body or furniture roots, type List[GroupItem]
+groups: [] # The parent + children relations capture nesting and reading-order.
+
+# All elements that have a text-string representation, type TextItem.
+# This is a flat list of all elements without implied order.
+texts:
+  - orig: "arXiv:2206.01062v1 [cs.CV] 2 Jun 2022"
+    text: "arXiv:2206.01062v1 [cs.CV] 2 Jun 2022"
+    self_ref: "#/texts/0"
+    label: "page_header"
+    parent:
+      $ref: "#/furniture"
+    children: []
```

Comment:
@cau-git I see something that may be an inconsistency:
- the parent has the `#` prefix (fragment separator for [URI fragment](https://datatracker.ietf.org/doc/html/rfc6901#section-6)): `$ref: "#/furniture"`, while
- the children do not have this prefix, e.g. `- $ref: "/texts/1"`

Can you clarify?

Reply:
Well spotted. The children must have it too.

Reply:
@vagenas Can you do the update?

Reply:
The fix needed is only to correct manually this file, since it was hand-crafted.
We should also consider validating the `ref` property to ensure it always follows the `#{}/{}` pattern.

Reply:
@PeterStaar-IBM @cau-git
1. validation added (which individually as expected causes the tests to fail due to the invalid data) and
2. test data fixed